File: LeetCode/1812.py
#Link: https://leetcode.com/problems/determine-color-of-a-chessboard-square/

class Solution:
    def squareIsWhite(self, coordinates: str) -> bool:
        x = ord(coordinates[0])-96
        y = int(coordinates[1])
        return (x+y)%2
        

File: LeetCode/cpp/1812.cpp
//Link: https://leetcode.com/problems/determine-color-of-a-chessboard-square/

class Solution {
public:
    bool squareIsWhite(string coordinates) {
        int x = coordinates[0]-96;
        int y = coordinates[1]-48;
        return ((x+y)%2 == 1) ? true : false;
    }
};
